api/cmd/main.go

```diff
@@ -42,8 +46,10 @@ func main() {
 	db.LogMode(false)
 	defer db.Close()
 
+	runDBMigration(db, cfg.DbConfig.MigrationPath)
```

Comment:
Since now the process of the DB initialization includes running DB migrations, can I ask you to move into a separate package, similar to how it's done in Turing: https://github.com/caraml-dev/turing/blob/dd88358d93d3665a6c7fa06c3d031ff7e99190dd/api/turing/database/database.go#L53?

`main()` should only call `db, err := database.InitDB(cfg.DbConfig)`, the rest should be incapsulated in a separate package to reduce the complexity of `main()` and to improve its maintainability.

Reply:
done